STYLEGUIDE.md

```diff
@@ -0,0 +1,101 @@
+- Heading case: Page titles (L1 headings) should use Title Case, all others should use Sentence case
+- 1 L1 Heading for page titles: Use ``######`` with overline; on first line of file; keep to a few words
+- L2 Headings for section heads: Use ========= with overline; 3 blank lines before them, will appear in sidebar so keep them short
+- L3 headings for subheads: Use ~~~~~~, 2 blank lines,
+- L4 Headings for sub-subheads if truly necessary: Use ---------, 2 blank lines, keep to a minimum (consider a flatter structure or breaking into multiple files)
+- Related components at the bottom: L3 heading, bullets, 2 blank lines before, try for 2-4 links if included
+- All files with Related components should contain at least one L2 section; if only one, can be  entitled Overview
+- Typically, for files with multiple sections or longer than one page, a summary section of a few lines length is helpful before the first section, summarizing the functionality discussed in the section and its purpose
+- Name of component/topic should be in first sentence, Title Case, bold on first use
+
+
+## Technical
+
+- Markdown, Github-flavored (``.md``) for any non-documentation rendered text files in the root of the repo, e.g. ``README``, ``CONTRIBUTING``, etc.
+- reStructuredText, (reST, ``.rst``) for documentation format, i.e. any user-visible files in the ``/doc`` directory
+- Where not otherwise contradicted by this guide or specific individual requirements, a subset of Markdown corresponding to that cross-compatible with reST should be preferred where possible; e.g. ``*italic*`` vs. ``_italic_``, ``*`` for bulleted lists, double backtick for inline code vs. single, etc.
+- Documentation should build with no warnings with ``Sphinx`` in nitpicky mode
+- Lower case filenames, rst extension, hyphens as space deliminators, short but descriptive
+- End files with a newline (blank line)
+- 1 blank line after all headings and before and after paragraphs, directives and "|"s
+- Use Line Feed (LF, ``\n``) as the End of Line character on all platforms
+- Use UTF-8 for all files (with no BOM, as dictated by the UTF-8 standard); do not use Windows Notepad as this adds a BOM and can otherwise corrupt files easily
+- Indicate special characters (e.g. em dash, en dash, etc) by simply including them inline with Unicode
+- Suggested to avoid non-ASCII characters where minimal downside/need exists, and recommended to keep used characters within the Latin-1 set where an additional range is not required by the specific usage, or an existing proper name or other identifier uses them
+- Spaces, no tabs for indentation and otherwise
+- Natural indents (3 spaces) for directives, 4 spaces for further indents beyond that in code
```

Comment:
Because that's the "natural" visual indent level and matches most other sources and style guides. It can be changed later, but its what's implemented now repo-wide.